src/schema/genre/genre.typedefs.ts

```diff
@@ -0,0 +1,19 @@
+import gql from 'graphql-tag';
+
+export const genreTypeDefs = gql`
+	type MovieInGenre {
+		id: ID!
+	}
+
+	type Genre {
+		id: ID!
+		title: String
+		movies: [MovieInGenre]
+	}
+
+	type Query {
+		"Genre: GET"
+		genre(id: ID!): Genre
+		genres(page: Int, limit: Int): [Genre!]
```

Comment:
Go rewatch my training from the other week! You need to do the return type differently. We want it to be a `GenreConnection`. You cannot paginate this result properly.

Reply:
Pagination added to gql:
![Screenshot 2023-11-21 at 10 55 43 AM](https://github.com/thisdot/movies-api/assets/22064697/e9d0c617-2664-4ba1-a585-349a69301b56)